gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsJdbcMonitor.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.util.concurrent.SimpleTimeLimiter;
 import io.airlift.log.Logger;
+import io.airlift.units.Duration;
```

Comment:
> Add a timeout for the backend stats collection query.

Remove the following period and wrap the commit body at 72 characters.
Please take a look at https://trino.io/development/process#pull-request-and-commit-guidelines-